### PR TITLE
(+) test correctness of h2bl matrices for gcov metric

### PR DESCRIPTION
**Pull request by korobkin**

### CODE REVIEW

File: script/analysis/check_transformation_matrices.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# Functions:
+# - print_matrix
+# - check_transformation_matrices
+# 
+
+from math import *
+import numpy as np
+
+def print_matrix(matrix,fmt="%19.11e",tostdout=True) -> str:
+  """Pretty-prints a matrix to a string (optinally, to stdout)
```
Comment (Yurlungur) [inline script/analysis/check_transformation_matrices.py:32]: Very nicely documented! That's appreciated!

**Review by Yurlungur — APPROVED** [review]
This is much appreciated! I think its fine as a utility script, and I'm happy to merge it in now. But we could also integrate it into the regression test suite.

We could merge this in now, and then if you're willing, I could walk you through the process of adding a regression test in a separate PR. What do you think?

**Comment by korobkin** [comment]
> We could merge this in now, and then if you're willing, I could walk you through the process of adding a regression test in a separate PR. What do you think?
That's right, it will certainly fit better in regression tests. Let's do it.

**Comment by Yurlungur** [comment]
Excellent. I'll merge this in now and let's talk this afternoon about how to do a regression test.

**Pull request merged**